lib/ReactViews/DataCatalog/DataCatalogGroup.jsx

```diff
@@ -72,11 +72,32 @@ const DataCatalogGroup = createReactClass({
             this.props.viewState.previewedItem === this.props.group;
     },
 
+    getNameOrPrettyUrl() {
+        // Grab a name via nameInCatalog, if it's a blank string, try and generate one from the url
+        const group = this.props.group;
+        const nameInCatalog = group.nameInCatalog || '';
+        if (nameInCatalog !== '') {
+            return nameInCatalog;
+        }
+
+        const url = group.url || '';
+        // strip protocol
+        const httpProtocol = url.indexOf('http');
```

Comment:
I think this would be simpler as a regular expression.
```suggestion
        return url.replace(/https?:\/\//, '')
```

Reply:
That it could, thanks!